src/client/pages/OfferNew/InsuranceSelector/Selector.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Selector } from './Selector'
+
+const mockInsurances = [
+  {
+    id: '1',
+    name: 'Inbo, Ulykke & Rejse',
+    price: 329,
+    currency: 'kr/md',
+    label: '3-in-1',
+    selected: true,
+  },
+  {
+    id: '2',
+    name: 'Inbo &  Ulykke',
+    price: 279,
+    currency: 'kr/md',
+    label: '2-in-1',
+    selected: false,
+  },
+  {
+    id: '3',
+    name: 'Inbo',
+    price: 189,
+    currency: 'kr/md',
+    selected: false,
+  },
+]
+
+export default {
+  title: 'Offer/InsuranceSelector',
+  component: Selector,
+  parameters: {
+    backgrounds: { default: 'gray100' },
+  },
+}
+
+export const Default = () => {
+  return <Selector insurances={mockInsurances} onChange={(id) => alert(id)} />
+}
```

Comment:
With these changes you can try it our better:

```suggestion
export const Default = () => {
  return <Selector insurances={mockInsurances} onChange={(id) => alert(id)} />
}export const Default = () => {
  const [insurances, setInsurances] = React.useState(mockInsurances)

  return (
    <Selector
      id="select-bundle"
      insurances={insurances}
      onChange={(id) =>
        setInsurances((prev) =>
          prev.map((item) => ({ ...item, selected: item.id === id })),
        )
      }
    />
  )
}
```